### PR TITLE
Select dom virtualisation

### PR DESCRIPTION
**Pull request by derpdead**
# Description

Implemented DOM virtualisation for Select kind components

Issue # (issue)

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Unit Tests
- [ ] e2e Test

# Checklist:

- [x] I have read the contribution requirements and fulfil them.
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by tomulus91 — APPROVED** [review]

File: modules/@ergonode/ui/src/components/Select/Dropdown/SelectDropdown.vue [hunk 1]
```diff
@@ -72,24 +78,30 @@
...
 } from '@Core/defaults/theme';
-import List from '@UI/components/List/List';
 import ClearSearchButton from '@UI/components/Select/Dropdown/Buttons/ClearSearchButton';
 import Dropdown from '@UI/components/Select/Dropdown/Dropdown';
 import MultiselectDropdownFooter from '@UI/components/Select/Dropdown/Footers/MultiselectDropdownFooter';
 import SelectDropdownApplyFooter from '@UI/components/Select/Dropdown/Footers/SelectDropdownApplyFooter';
 import DropdownListElement from '@UI/components/Select/Dropdown/List/DropdownListElement';
 import DropdownPlaceholder from '@UI/components/Select/Dropdown/Placeholder/DropdownPlaceholder';
 import FadeTransition from '@UI/components/Transitions/FadeTransition';
+// import {
```
Comment (tomulus91) [inline modules/@ergonode/ui/src/components/Select/Dropdown/SelectDropdown.vue]: why not deleted ?

File: modules/@ergonode/ui/src/components/Select/Dropdown/SelectDropdown.vue [hunk 2]
```diff
@@ -72,24 +78,30 @@
...
+// } from 'vue-windowing';
+import Vue from 'vue';
+import ViewWindowing from 'vue-windowing';
+
+Vue.use(ViewWindowing);
 
 export default {
     name: 'SelectDropdown',
     components: {
+        // VirtualScroll,
```
Comment (tomulus91) [inline modules/@ergonode/ui/src/components/Select/Dropdown/SelectDropdown.vue]: why not deleted ?

File: modules/@ergonode/core/src/store/core/actions.js [hunk 3]
```diff
@@ -42,7 +42,7 @@ export default {
             if (this.app.$axios.isCancel(e)) {
                 return;
             }
-
+            console.log(e);
```
Comment (gygoo) [inline modules/@ergonode/core/src/store/core/actions.js:45]: rm -> console.log

File: modules/@ergonode/core/src/components/Tabs/MainSettingsTab.vue [hunk 4]
```diff
@@ -108,7 +108,7 @@ export default {
             });
         },
         async onUpdateSuccess() {
-            await this.getLanguages();
+            await this.getLanguages({});
```
Comment (gygoo) [inline modules/@ergonode/core/src/components/Tabs/MainSettingsTab.vue:111]: maybe better to set default value in function?

**Pull request closed without merging**